st2common/st2common/util/param.py

```diff
@@ -88,16 +88,22 @@ def _process(G, name, value):
     if isinstance(value, str):
         value = to_unicode(value)
     template_ast = ENV.parse(value)
+    LOG.debug('Template ast: %s', template_ast)
     # Dependencies of the node represent jinja variables used in the template
     # We're connecting nodes with an edge for every depencency to traverse them in the right order
     # and also make sure that we don't have missing or cyclic dependencies upfront.
     dependencies = meta.find_undeclared_variables(template_ast)
+    LOG.debug('Dependencies: %s', dependencies)
     if dependencies:
```

Comment:
Turns out that refactoring https://gist.github.com/lakshmi-kannan/24fcbf5bcd05b81b765e6665af317ada breaks the tests :/.